srcpkgs/leiningen/template

```diff
@@ -1,24 +1,28 @@
 # Template file for 'leiningen'
 pkgname=leiningen
-version=2.9.5
+version=2.10.0
 revision=1
 depends="bash virtual?java-runtime"
 short_desc="Automate Clojure projects without setting your hair on fire"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Adam Frey <adam@adamfrey.me>"
 license="EPL-2.0"
 homepage="http://leiningen.org/"
-distfiles="https://github.com/technomancy/${pkgname}/archive/${version}.tar.gz
- https://github.com/technomancy/${pkgname}/releases/download/${version}/leiningen-${version}-standalone.zip"
-checksum="a29b45966e5cc1a37d5dc07fe436ed7cb172c88c53d44a049956ff53a096d43e
- df490c98bfe8d667bc5d83b80238528877234c285d0d48f61a4c8743c2db1eea"
-skip_extraction="leiningen-${version}-standalone.zip"
+distfiles="https://codeberg.org/${pkgname}/${pkgname}/archive/${version}.tar.gz
+ https://codeberg.org/attachments/${jar_id}"
+checksum="6b04dbe0934ab4c34dc27b0055ad290d2ba9c0f454c1bda7defa58d62946b743
+ d27299bad34075ac2864d0bd0559f835c6e2c476c0b0a283bcbdb574fdadbb34"
+_jar_id=43cebda5-a7c2-405b-b641-5143a00051b5
+
+do_extract() {
+	bsdtar xf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${version}.tar.gz
+}
```

Comment:
why is this required? skip_extraction is not used

Reply:
Oops, I was confused about the `skip_extraction` functionality. I force-pushed [a new commit](https://github.com/void-linux/void-packages/pull/41825/commits/aac814bfdaa78cf1a4585b4975a1c9cc4573aa1f) which avoids `do_extract`.